### DISCUSSION TITLE
Preview "Help" page is not reloaded after first `devtools::document()`

### DISCUSSION THREAD

**Discussion by etiennebacher** [posit-dev/positron#5559] [Q&A]
Hello, when doing package dev, I noticed that the preview "Help" page is not reloaded after the first `devtools::document()`. In the example below, after changing the title in the `roxygen2` content, the preview page doesn't reload and still shows the original "A title". The only workaround I have so far is to close and reopen Positron, which is of course not ideal.

[Screencast from 2024-11-28 14-16-44.webm](https://github.com/user-attachments/assets/cc1529c5-4597-4a69-906d-15c950de289b)

**Comment by juliasilge** [comment]
Thank you for reporting this! 🙌  I opened #5568 to track and prioritize this bug.
